packages/block-editor/src/hooks/align.js

```diff
@@ -155,54 +154,27 @@ function BlockEditAlignmentToolbarControlsPure( {
 export default {
 	shareWithChildBlocks: true,
 	edit: BlockEditAlignmentToolbarControlsPure,
+	useBlockProps,
```

Comment:
Maybe useBlockEditProps, I can see us having useBlockSaveProps, or useBlockProps for both?

Reply:
Maybe we should keep it consistent with the block API? `useBlockProps` and `useBlockProps.save`? One is a hook and the other one is static.

Reply:
But yes, it's not entirely the same so maybe a different naming is appropriate.